exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillResultSetImpl.java

```diff
@@ -850,9 +851,9 @@ public void moveToCurrentRow() throws SQLException {
   }
 
   @Override
-  public DrillStatementImpl getStatement() {
+  public AvaticaStatement getStatement() {
```

Comment:
Any specific reason to have this method override it's parent implementation ? it's basically doing the same thing